docs/getting_started.rst

```diff
@@ -41,7 +41,8 @@ Pinot requires JDK 8 or later and Apache Maven 3.
 
 #. Check out the code from GitHub (https://github.com/apache/incubator-pinot)
 #. With Maven installed, run ``mvn install package -DskipTests -Pbin-dist`` in the directory in which you checked out Pinot.
-#. Make the generated scripts executable ``cd pinot-distribution/target/apache-pinot-incubating-<version>-SNAPSHOT-bin; chmod +x bin/*.sh``
+#. Set pinot version variable ``export PINOT_VERSION=0.2.0``
```

Comment:
It will be great if this can be set automatically - can this be pulled from the parent pom file?